test/plugins/scenarios/trace-activation/test_service.go

```diff
@@ -112,6 +118,7 @@ func testAsyncInCrossGoroutine() {
 	go func() {
 		s.SetTag("testAsyncTag", "success")
 		s.AddLog("testAsyncLog", "success")
+		s.AddEvent(trace.WarnEventType, "bar")
```

Comment:
If you have added the event in the async mode, why the test case cannot found this event?

Reply:
You can see it in the logEvent  of the `operationName: testAsyncInCrossGoroutine` test case  :)